frontend/src/pages/org/workflow-detail.ts

```diff
@@ -616,12 +616,37 @@ export class WorkflowDetail extends BtrixElement {
     const workflow = this.workflow;
 
     const archivingDisabled = isArchivingDisabled(this.org, true);
+    const paused = workflow.lastCrawlState === "paused";
+    const hidePause =
+      !this.lastCrawlId ||
+      this.isCancelingOrStoppingCrawl ||
+      this.workflow.lastCrawlStopping;
+    const disablePause =
+      this.workflow.lastCrawlPausing ===
+      (this.workflow.lastCrawlState === "running");
```

Comment:
Should this just be checking `lastCrawlPausing`?
```suggestion
    const disablePause = this.workflow.lastCrawlPausing;
```

Reply:
I think the idea was that the button is both Pause / Resume, and possible states are:
- pausing and running (disable, already in process of Pausing)
- pausing and not running (don't disable, it should show Resume)
- not pausing and not running (disable, already in process of Resuming)
- not pausing and running (don't disable, it should show Pause)

Had to double check that, perhaps a bit more confusing then it needs to be.. maybe should rename it to disablePauseResume to be more clear

Reply:
The `lastCrawlPausing` / `pausing` reflects if user requested crawl to be paused/not paused, while crawl state is the actual state the crawl is in, so we enable the button when they are different.

Reply:
Yeah a variable rename would work, `hidePauseResume` and `disablePauseResume`.

Is there any need to add and check for `lastCrawlResuming`?

Reply:
> Yeah a variable rename would work, `hidePauseResume` and `disablePauseResume`.
> 
> Is there any need to add and check for `lastCrawlResuming`?

Resuming is the when lastCrawlPausing is set to false. I suppose it could be renamed too - if true, it means the user wants the crawl paused, if false, it means the user wants the crawl running/resumed (the default).

Reply:
Updated this to hidePauseResume and disablePauseResume. Also did a pass to replace instance of 'unpause' with 'resume', including in the API, to be consistent.